tests/std/test.lst

```diff
@@ -239,6 +239,7 @@ tests\GH_004040_container_nonmember_functions
 tests\GH_004109_iter_value_t_direct_initialization
 tests\GH_004201_chrono_formatter
 tests\GH_004275_seeking_fancy_iterators
+tests\GH_0004388_unordered_meow_operator_equal

```

Comment:
Sorry if I missed it, but did we start using 7-digit numbers for GH issues? If not, this has an extra leading "0" that makes it not lexical-sorted.